setup.py

```diff
@@ -75,6 +75,7 @@ def run_tests(self):
         "Mako",
         "parse",
         "parse_type",
+        "py",
```

Comment:
should we try to remove it then completely in this PR?

Reply:
the only thing left is `py.io.TerminalWriter` -- which even `pytest` still uses :( (to be factored out eventually as `py` is phased out)